fe/fe-core/src/main/java/com/starrocks/analysis/ModifyFrontendAddressClause.java

```diff
@@ -0,0 +1,38 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.analysis;
+
+import com.starrocks.ha.FrontendNodeType;
+import com.starrocks.sql.ast.AstVisitor;
+
+
+public class ModifyFrontendAddressClause extends FrontendClause {
+
+    protected String toBeModifiedHost;
```

Comment:
protected String srcHost;
protected String destHost;//destHost must be in an FQDN format.